karma.conf.cjs

```diff
@@ -8,11 +8,12 @@ module.exports = function(config) {
 
     // frameworks to use
     // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
-    frameworks: ["jasmine"],
+    frameworks: ["parallel", "jasmine"],
```

Comment:
The need to run tests in separate browsers/tabs is because styled-components is absolutely required to be a singleton instance. If these were run in the same instance, you would get the following warning: 

```
WARN: 'It looks like there are several instances of 'styled-components' initialized in this application. This may cause dynamic styles not rendering properly, errors happening during rehydration process, missing theme prop, and makes your application bigger without a good reason.
```

This error is the entire reason for my motivation for this package. In single-spa, styled-components should be a shared dependency and this package makes that simpler because it is a system module.

Reply:
Good to know 👍